### ISSUE TITLE
Different behaviour than 3.0.0 MR: CoilSensitivityMap wants sorted data

### ISSUE THREAD

**Issue by paskino** [SyneRBI/SIRF#957]
This code block from the introductory notebook on [PET/MR/CT](https://github.com/SyneRBI/SIRF-Exercises/blob/master/notebooks/Introductory/acquisition_model_mr_pet_ct.ipynb) has changed behaviour with one of the latest commits and requires to sort the `preprocessed_data` before passing it to `CoilSensitivityData` object.

```python
# 1. create MR AcquisitionData
mr_acq = mr.AcquisitionData(os.path.join(examples_data_path('MR'),'grappa2_1rep.h5'))

# 2. calculate CSM
preprocessed_data = mr.preprocess_acquisition_data(mr_acq)
##########################
# THIS WAS NOT NEEDED BEFORE
preprocessed_data.sort()
##########################
csm = mr.CoilSensitivityData()
csm.smoothness = 50
csm.calculate(preprocessed_data)
```

What has changed?

**Comment by KrisThielemans** [comment]
> What has changed?

a lot!

However, will be fixed by #950

**Comment by paskino** [comment]
You tell me! Nothing seemed to behave with all the changes that had been merged!
